third_party/mbedtls/mbedtls-config.h

```diff
@@ -26,7 +26,8 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef MBEDTLS_CONFIG_H
+#ifndef \
```

Comment:
add comment here to explain why this is necessary?

Reply:
Done.